compute_tools/src/config.rs

```diff
@@ -93,5 +93,25 @@ pub fn write_postgres_conf(
         writeln!(file, "neon.extension_server_port={}", port)?;
     }
 
+    // This is essential to keep this line at the end of the file,
+    // because it is intended to override any settings above.
+    writeln!(file, "include_if_exists = 'compute_ctl_temp_override.conf'")?;
```

Comment:
Not sure what is better, but I thought about just appending `neon.max_cluster_size=-1` to the config before start, then doing a full `config::write_postgres_conf()` again to override it.

That way we don't leave any dangling `include_if_exists` after start. Will be also a bit less code, probably

Reply:
I wanted to change `postgresq.conf` as less as possible.

plus, `Include_if_exists` reminds that the config was different during compute_ctl.
otherwise there will be no trace of this override magic at all...